ContactApp3/View Controllers/ContactsListViewController.swift

```diff
@@ -86,5 +89,25 @@ extension ContactsListViewController: UITableViewDelegate, UITableViewDataSource
             self.navigationController?.pushViewController(vc, animated: true)
         }
     }
+    
+    func tableView(_ tableView: UITableView, leadingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        
+        //Creat swipe action
```

Comment:
@oulamardawi combine these two lines together (join declaration and assignment) of action